sugargame/canvas.py

```diff
@@ -1,51 +1,57 @@
 import os
 from gi.repository import Gtk
-from gi.repository import GLib
+from gi.repository import GObject
 import pygame
 import event
 
 CANVAS = None
 
 class PygameCanvas(Gtk.EventBox):
-    
+
     """
     mainwindow is the activity intself.
     """
     def __init__(self, mainwindow, pointer_hint = True):
-        Gtk.EventBox.__init__(self)
+        GObject.GObject.__init__(self)
 
         global CANVAS
         assert CANVAS == None, "Only one PygameCanvas can be created, ever."
         CANVAS = self
 
         # Initialize Events translator before widget gets "realized".
         self.translator = event.Translator(mainwindow, self)
-        
+
         self._mainwindow = mainwindow
 
         self.set_can_focus(True)
-        
+
         self._socket = Gtk.Socket()
         self.add(self._socket)
+
+        self._initialized = False
```

Comment:
I've since removed this `self._initialized` in Sugargame, can you use the latest sources please?